src/services/conseillers/conseillers.class.js

```diff
@@ -940,6 +946,55 @@ exports.Conseillers = class Conseillers extends Service {
       }
       res.send({ disponible });
     });
+
+    app.patch('/conseillers/update_date_disponibilite/:id', async (req, res) => {
+      checkAuth(req, res);
+      const accessToken = req.feathers?.authentication?.accessToken;
+      const userId = decode(accessToken).sub;
+      const user = await db.collection('users').findOne({ _id: new ObjectId(userId) });
+      const idConseiller = req.params.id;
+      const { dateDisponibilite } = req.body;
+
+      const dateDisponibleValidation =
+        Joi.date().error(new Error('La date est invalide, veuillez choisir une date supérieure ou égale à la date du jour')).validate(dateDisponibilite);
+      if (dateDisponibleValidation.error) {
+        res.status(400).json(new BadRequest(dateDisponibleValidation.error));
+        return;
+      }
+      const conseiller = await db.collection('conseillers').findOne({ _id: new ObjectId(idConseiller) });
+      if (!conseiller) {
+        res.status(404).send(new NotFound('Ce conseiller n\'existe pas', {
+          idConseiller,
+        }).toJSON());
+        return;
+      }
+      if (String(conseiller._id) !== String(user.entity.oid)) {
+        res.status(403).send(new Forbidden('Action non autorisée', {
+          userId
+        }).toJSON());
+        return;
+      }
+      try {
+        await pool.query(`UPDATE djapp_coach
+            SET start_date = $2 WHERE id = $1`,
+        [conseiller.idPG, dateDisponibilite]);
```

Comment:
updatedAt ?

Reply:
Il n'est présent dans aucune requête pg

Reply:
Oui et c'est un souci je pense dans notre cas  : src\tools\migration\migrate-pg-to-mongo.js 
Dans la synchro PG -> Mongo, il y a ce champ updatedAt qui est remis à jour.  Si on le met à jour que côté Mongo, il se fera réécraser